tests/ClosureDefaultSerializerTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Spatie\Fractal\Test;
+
+use Spatie\Fractalistic\ArraySerializer;
+
+class ClosureDefaultSerializerTest extends TestCase
+{
+    public function setUp($defaultSerializer = '')
+    {
+        parent::setup(function () {
+            return new ArraySerializer();
+        });
+    }
+
+    /** @test */
+    public function it_uses_the_default_transformer_when_it_is_specified()
```

Comment:
I'd personally rename this to `it_uses_configured_transformer_when_toarray_is_used `.